src/wallet/coinselection.cpp

```diff
@@ -41,10 +41,10 @@ CAmount reasonableExcess(CAmount targetValue)
 }
 
 // Create a group of Txos from a Txo index and another group.
-TxoGroup makeGroup(SpendableTxos::iterator i, const TxoGroup& prev = *(TxoGroup*)NULL)
+TxoGroup makeGroup(SpendableTxos::iterator i, const TxoGroup* prev = NULL)
 {
   TxoGroup ret;
-  if (&prev) ret = prev;
+  if (prev != NULL) ret = *prev;
   else ret.first = 0;
```

Comment:
use C++11 nullptr

Reply:
there really is no difference in this case since the function is setting it to NULL in the default parameter.  

although ptschip does make a good point that using nullptr is probably better in the future over NULL is better because you cant &nullptr, it wont compile. probably better for the project style guide to demand the use of nullptr.

Reply:
@ptschip : Alright, I changed `NULL` to `nullptr`.